framework/wazuh/core/cluster/dapi/dapi.py

```diff
@@ -32,10 +32,22 @@
 from wazuh.core.exception import WazuhException, WazuhClusterError, WazuhError
 from wazuh.core.wazuh_socket import wazuh_sendsync
 
-pools = common.mp_pools.get()
 
 authentication_funcs = {'check_token', 'check_user_master', 'get_permissions', 'get_security_conf'}
-events_funcs = {"send_event_to_analysisd"}
+events_funcs = {'send_event_to_analysisd'}
+
+node_info = wazuh.core.cluster.cluster.get_node()
+if node_info['type'] == 'master':
```

Comment:
Previously, we were creating the pool in the worker nodes as well, which is not necessary because they don't execute authentication requests.

Reply:
All authentication requests are `local_master`. The only case in which `execute_local_request` is run in a worker node is when either the DAPI or the cluster are disabled, in which case we could expect the only running node to be the master, because workers don't have a `rbac.db` and can't handle auth requests.

I think we could avoid creating the processes on worker nodes but I would understand if we decide to go with a same path and leave it as it was before.